common/src/main/scala/io/phdata/generators/JsonTopicGenerator.scala

```diff
@@ -10,30 +10,30 @@ import org.fusesource.scalate.TemplateEngine
 
 //TODO: LDAPGroupGenerator can be removed when we switch to this topic generator, now it's here only for backward compatibility
 class JsonTopicGenerator[F[_]](appConfig: AppConfig, ldapGroupGenerator: LDAPGroupGenerator[F])(
-  implicit clock: Clock[F],
-  F: Sync[F]
+    implicit clock: Clock[F],
+    F: Sync[F]
 ) extends TopicGenerator[F] with LazyLogging {
 
   val templateEngine = new TemplateEngine()
 
   override def topicFor(
-                         name: String,
-                         partitions: Int,
-                         replicationFactor: Int,
-                         workspaceRequest: WorkspaceRequest
-                       ): F[KafkaTopic] = {
+      name: String,
+      partitions: Int,
+      replicationFactor: Int,
+      workspaceRequest: WorkspaceRequest
+  ): F[KafkaTopic] = {
     val templatePath = Paths.get(appConfig.templates.templateRoot, "topic.ssp")
 
     generateTopic(name, partitions, replicationFactor, workspaceRequest, templatePath.toString)
   }
 
   private def generateTopic(
-                             name: String,
```

Comment:
Did you run scalafmt on this?

Reply:
@afoerster I ran scalafmt before I pushed the commit.